crates/starknet/src/error.rs

```diff
@@ -14,8 +14,8 @@ pub enum Error {
     BlockifierStateError(#[from] blockifier::state::errors::StateError),
     #[error(transparent)]
     BlockifierTransactionError(#[from] blockifier::transaction::errors::TransactionExecutionError),
-    #[error("Failed execution during fee estimation: {msg}")]
-    EstimationExecutionError { msg: String },
+    #[error("{revert_error}")]
+    ExecutionError { revert_error: String },
```

Comment:
I approved the PR, but now I remember, that I deliberately named this `EstimationExecutionError` because there is also `TransactionExecutionError`, so to distinguish that this one can originate only from estimation.

Reply:
Why this error variany cant be used in other places

Reply:
I guess it can, but then we have two overlapping errors (this `ExecutionError` and the old `TransactionExecutionError` in crates/types/src/error.rs)

Reply:
The difference between the two is that ExecutionError accepts string as a parameter, but TransactionExecutionError array of Felts. Its easier to use our implementation of ExecutionError, due to the above mentioned